src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -70,8 +70,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
             educationVo.setYear(inquiryEducation.get(i).getYear() == null ? null : inquiryEducation.get(i).getYear());
             educationVo.setType(inquiryEducation.get(i).getType() == null ? null : inquiryEducation.get(i).getType());
             iterator.next();
-            i++;
             educationVos.add(educationVo);
+            i++;
```

Comment:
Can we use for each loops instead?

Reply:
yes, I have tried it in commit 8da9452 please review